types/japanese-holidays/index.d.ts

```diff
@@ -3,10 +3,33 @@
 // Definitions by: syamatoo <https://github.com/syamatoo>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export as namespace JapaneseHolidays;
+
 export function isHoliday(date: Date, furikae?: boolean): string | undefined;
 export function isHolidayAt(date: Date, furikae?: boolean): string | undefined;
 export function getHolidaysOf(year: number, furikae?: boolean): Holiday[];
```

Comment:
Ah, but actually I used `year` instead of `y` because document is `year` but implementation is `y` :)
Then why don't we use `date` instead of `d` for all of `Date` arguments?

Reply:
Done, amended. `month` used, not `mon` (used in both form in the source), thx!